include/SDL3/SDL_iostream.h

```diff
@@ -308,6 +308,8 @@ extern SDL_DECLSPEC SDL_IOStream * SDLCALL SDL_IOFromFile(const char *file, cons
  */
 extern SDL_DECLSPEC SDL_IOStream * SDLCALL SDL_IOFromMem(void *mem, size_t size);
 
+#define SDL_PROP_IOSTREAM_MEM_BASE_POINTER "SDL.iostream.mem.base"
```

Comment:
Needs documentation. Also, you should add `SDL_PROP_IOSTREAM_MEM_SIZE_NUMBER`

Reply:
Maybe this should be named `SDL_PROP_IOSTREAM_STATIC_MEMORY_POINTER` for consistency with `SDL_PROP_IOSTREAM_DYNAMIC_MEMORY_POINTER`?

Reply:
I do wonder if there's a good word for this type of pointer - 'static' isn't as straightforward as you'd hope since this is often being used to read something dynamically allocated elsewhere. Also can't use 'constant' since there's the const/non-const variants... not sure!

Reply:
Works for me - if nobody beats me to it I'll push this early tomorrow!